.github/workflows/gen-scie-platforms.yml

```diff
@@ -71,7 +75,7 @@ jobs:
             -v $PWD:/code \
             -w /code \
             --platform linux/${{ matrix.docker-arch }} \
-            python:3.11-slim-bullseye bash ./gen-scie-platform.sh
+            python:3.11-slim-bookworm bash ./gen-scie-platform.sh
```

Comment:
Had to update to bookworm, as cryptography was compled with OpenSSL 3

Reply:
SGTM